packages/cli/src/__tests__/commands/CLI.test.ts

```diff
@@ -95,7 +95,7 @@ it('introspect should include deprecation warning', async () => {
 
   await expect(result).rejects.toMatchInlineSnapshot(`
           Could not find a schema.prisma file that is required for this command.
-          You can either provide it with --schema, set it as \`prisma.schema\` in your package.json or put it into the default location ./prisma/schema.prisma https://pris.ly/d/prisma-schema-location
+          You can either provide it with --schema, set it as \`schema.prisma\` in your package.json or put it into the default location ./prisma/schema.prisma https://pris.ly/d/prisma-schema-location
```

Comment:
I think in this case `prisma.schema` was correct since it's talking about `pacakge.json` key, not the file.

Reply:
Ouch, good point!